fastlane/lib/fastlane/actions/verify_build.rb

```diff
@@ -85,7 +85,7 @@ def self.print_values(values)
             columns << key
             columns << case value
                        when Hash
-                         value.map {|k, v| "#{k}: #{v}"}.join("\n")
+                         value.map { |k, v| "#{k}: #{v}" }.join("\n")
```

Comment:
And this closure got a trailing one. It feels like something went a bit wrong with the first ones I mentioned above.